ocpairprogramming/src/main/java/com/pino/project/ocpairprogramming/java8/ocp/chapter3/generics/MyStackOutOfLinkedList.java

```diff
@@ -0,0 +1,30 @@
+package com.pino.project.ocpairprogramming.java8.ocp.chapter3.generics;
```

Comment:
we can write other ways to implement a stack through a LinkedList, based on the fact that implements Queue.
Maybe in a MyStackOutOfQueue class.

Reply:
Yes. Please go ahead by adding whatever you retain useful into your PR